### DISCUSSION TITLE
[Suggestion] Snow Layers Placed by Players will not Melt

### DISCUSSION THREAD

**Discussion by Munchkinnibbler** [PurpurMC/Purpur#544] [__Dismissed__] [answered]
As the title suggests, it would be nice for snow layers that are placed by players to not melt if it is possible.

**Comment by granny** [comment, marked as answer]
Would be better as a plugin that cancels the [BlockFadeEvent](https://purpurmc.org/javadoc/org/bukkit/event/block/BlockFadeEvent.html) event. Transferring this over to the PurpurExtras plugin. https://github.com/PurpurMC/PurpurExtras/issues/65
